Vic2ToHoI4/Source/HOI4World/HoI4Country.h

```diff
@@ -313,6 +331,7 @@ class Country
 
 	std::string tag;
 	std::string oldTag;
+	std::string originalTag;
```

Comment:
Is this a different concept than the oldTag? If not, combine them. If so, they probably need better names.